### PR TITLE
Change way adding nav entry to allow l10n

### LINKED ISSUE #863: french locale shows both Market and Marché
Seen in 10.8.0

image

expected behaviour: 

The translation Marché should be used everyhere.

### PR DESCRIPTION
**Pull request by JanAckermann**
Fixes #863

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -25,8 +25,4 @@ To use this application click on "Files" in the top left corner and click on "Ma
 		<command>OCA\Market\Command\ListApps</command>
 		<command>OCA\Market\Command\UpgradeApp</command>
 	</commands>
-	<navigation role="admin">
```
Comment (DeepDiver1975) [inline appinfo/info.xml:28]: Why was that removed?

File: l10n/Makefile [hunk 2]
```diff
@@ -2,11 +2,11 @@ NODE_BINDIR = ../node_modules/.bin
 export PATH := $(NODE_BINDIR):$(PATH)
 
 # Available locales for the app.
-LOCALES = af_ZA  bg_BG  ca  cs_CZ  da  de  de_DE  el  en_GB  es  et_EE  fi_FI  fr  he  hu_HU  is  it  ja  ko  lt_LT  nb_NO  nl  oc  pl  pt_BR  pt_PT  ru  sl  sq  sv  th_TH  tr  uk  zh_CN
```
Comment (DeepDiver1975) [inline l10n/Makefile:5]: oc no longer on transifex I assume?

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

File: appinfo/info.xml (hunk 1 above)
Comment (JanAckermann) [reply appinfo/info.xml:28]: Added here: lib/Application.php to allow translations

File: l10n/Makefile (hunk 2 above)
Comment (JanAckermann) [reply l10n/Makefile:5]: we don't have min percentage of 15% translations here (so .po file won't be pulled), therefore the command `gettext-compile --output translations.json $(LOCALE_FILES)` fails

File: appinfo/info.xml (hunk 1 above)
Comment (JanAckermann) [reply appinfo/info.xml:28]: Let me revert this and check if it works now, l10n should be there now =)

**Review by DeepDiver1975 — APPROVED** [review]

**Pull request merged**